xonsh/environ.py

```diff
@@ -1231,6 +1203,8 @@ def partial_format_prompt(template=DEFAULT_PROMPT, formatter_dict=None):
     except:
         return _failover_template_format(template)
 
+format_prompt = partial_format_prompt
```

Comment:
@scopatz  Maybe `partial_format_prompt` should be renamed to, and replace, `format_prompt`?